api/producer/firehose/tests/conftest.py

```diff
@@ -0,0 +1,92 @@
+import uuid
+
+import pytest
+
+from api.producer.firehose.tests.e2e_utils import (
+    make_good_cloudwatch_data,
+    submit_cloudwatch_data_to_firehose,
+)
+from helpers.aws_session import new_aws_session
+from helpers.terraform import get_terraform_json
+
+
+@pytest.fixture(scope="session")
+def global_event_handler():
+    return {}
+
+
+@pytest.fixture(scope="session")
+def session():
+    return new_aws_session()
+
+
+@pytest.fixture(scope="session")
+def stream_arn():
+    firehose_metadata = get_terraform_json()["firehose"]["value"]["producer"][
```

Comment:
Im assuming there is no way to reduce the repetition of this line through the next 4 functions? Minor thing though as I cant think of something that wouldnt make it more complicated haha

Reply:
i could write another function from which these fixtures draw from, but i think it would have little value in terms of readability. These wouldn't be necessary if I wasn't using fixtures for these constants, but I'm only using them since it makes the pytest itself a whole lot more readable with a trade off that there are magic fixtures sitting in the conftest. Bit of a tricky one